yarn.lock

```diff
@@ -0,0 +1,903 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Delete me 🗑️

Reply:
Seems like the file is autogenerated and it gets pushed along with all the other changes despite not being staged. 
I will pay attention to deleting it before making any commits from now on.

Reply:
As an alternative, you can ignore the file on your local clone. To do that, add `yarn.lock` on a new line in `.git/info/exclude` (i.e., from the root of the repository, run `echo 'yarn.lock' >> .git/info/exclude`).